### PR TITLE
Improved the usability for a new user

### PR DESCRIPTION
**Pull request by ghost**
- Added a section on how to create an API key (previously missing) 
- Added details for creating a .py or .js file
- Clarified adding the code snippet from each step into the existing file
- Improved the steps for actually seeing messages published and subscribed in the 'full demo file'
- TO DO: Change the code block for the full demo file to work properly

### CODE REVIEW

File: docs/streams/getting-started-streams.md [hunk 1]
```diff
@@ -11,12 +11,15 @@ This article is an introduction to using streams with [Macrometa SDKs](../sdks/i
...
 
 - Access to a Macrometa account with sufficient permissions to create streams.
+- Have your API Key ready. For more information, refer to [Create API Keys] (../account-management/api-keys/create-api-keys)
 - Install the appropriate SDK. For more information, refer to [Install SDKs](../sdks/install-sdks.md).
 
 ## Get Started with Streams
 
 The basic parts of this quickstart walk you through creating a stream, publishing messages to it, and subscribing to the stream using the [pyC8](https://pyc8.readthedocs.io/en/latest/) and [jsC8](https://www.npmjs.com/package/jsc8) SDKs.
 
+To get started, create a new javascript (.js) or python (.py) file in your favorite IDE. Then, copy the code block below and paste it into your javascript or python file. With each subsequent step, append the code block to the existing file and run it.
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]:
```suggestion
To get started, create a new JavaScript (.js) or python (.py) file in your favorite IDE. Then, copy the code block below and paste it into your JavaScript or Python file. With each subsequent step, append the code block to the existing file and run it.
```

File: docs/streams/getting-started-streams.md [hunk 2]
```diff
@@ -58,7 +61,7 @@ warnings.filterwarnings("ignore")
 # Define constants
 URL = "gdn.paas.macrometa.io"
 GEO_FABRIC = "_system"
-API_KEY = "my API key" # Change this to your API key
+API_KEY = "my API key" # CHANGE THIS TO YOUR API KEY
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]: Please do not use all caps.

File: docs/streams/getting-started-streams.md [hunk 3]
```diff
@@ -267,7 +270,11 @@ for i in range(10):
 
 ## Full Demo File
 
-Copy the code block below into a file and run it in your favorite IDE.
+It's time to see streams in action! Replace the contents of your .js or .py file from above with the code block below. In your browser, open the GDN console and navigate to the 'streams' tab. Select your recently created stream (c8globals.streamQuickstart) to view the output of the message within the console.
+  - Open two terminal windows and start the program in each window 
+  - In one terminal, type 'r' to begin listening for messages, while in the other terminal, type 'w' to begin writing messages
+  - Upon each write, you should see the message received in the 2nd terminal window, as well as the message displayed in the GDN console output
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]:
```suggestion
  - Upon each write, you should see the message received in the second terminal window, as well as the message displayed in the GDN console output
```

File: docs/streams/getting-started-streams.md [hunk 4]
```diff
@@ -267,7 +270,11 @@ for i in range(10):
 
 ## Full Demo File
 
-Copy the code block below into a file and run it in your favorite IDE.
+It's time to see streams in action! Replace the contents of your .js or .py file from above with the code block below. In your browser, open the GDN console and navigate to the 'streams' tab. Select your recently created stream (c8globals.streamQuickstart) to view the output of the message within the console.
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]:
```suggestion
It's time to see streams in action! 

1. Replace the contents of your .js or .py file from above with the code block below.
2. In your browser, open the GDN console and then click **Streams**.
3. Select your recently created stream (c8globals.streamQuickstart) to view the output of the message within the console.
```

Comment (diana-macrometa) [reply docs/streams/getting-started-streams.md]: I didn't know you could do this! Neat!

File: docs/streams/getting-started-streams.md [hunk 5]
```diff
@@ -267,7 +270,11 @@ for i in range(10):
...
+  - Open two terminal windows and start the program in each window
```
Comment (diana-macrometa) [inline docs/streams/getting-started-streams.md]: Continue the numbered list above.

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
Great changes! I have a few suggestions, mostly for formatting. Please let me know if you have any questions about my suggestions.

File: docs/streams/getting-started-streams.md (hunk 1 above)
Comment (ghost) [reply docs/streams/getting-started-streams.md]: Looks good! We forgot to capitalize the first instance of the word 'python'

**Review by ghost — COMMENTED** [review]
Finalized all suggestions

**Review by diana-macrometa — APPROVED** [review]
LGTM!

**Pull request merged**